app/components/job_application_review_component/section.rb

```diff
@@ -21,10 +22,11 @@ def heading_text
     t("jobseekers.job_applications.build.#{@name}.heading")
   end
 
-  def build_list
-    list = nil
-    govuk_summary_list { |l| list = l }
-    list
+  def edit_link
+    title = heading_text
+    text = "Change"
+    href = error_path
+    govuk_link_to text, href, aria: { label: "#{text} #{title}" }, classes: "govuk-!-display-none-print" if text && href
```

Comment:
If I read this right the variable declarations and conditional link are  redundant? `text` and `href` will always be present, right?

Also... the "change" link is hardcoded to an "error_path"? Is this intentional? I have the feeling I am missing a bigger picture than this little bit of code :sweat_smile: 

```suggestion
    govuk_link_to "Change", error_path, aria: { label: "#{text} #{heading_text}" }, classes: "govuk-!-display-none-print"
```

Reply:
error path isn't always set. This piece of code is old, its just re-located into here